packages/react/src/components/Button/index.story.tsx

```diff
@@ -21,167 +21,170 @@ export default {
       },
     },
   },
-}
-
-const DefaultStory = (args: ButtonProps) => (
-  <Button {...args} onClick={action('click')}>
-    Button
-  </Button>
-)
+  render: (args) => <Button {...args}>Button</Button>,
+} as Meta<typeof Button>
 
-export const Default: Story<ButtonProps> = DefaultStory.bind({})
+export const Default: StoryObj<typeof Button> = {}
 
-export const Primary: Story<ButtonProps> = DefaultStory.bind({})
-Primary.args = {
-  variant: 'Primary',
+export const Primary: StoryObj<typeof Button> = {
+  args: {
+    variant: 'Primary',
+  },
 }
 
-export const Navigation: Story<ButtonProps> = DefaultStory.bind({})
-Navigation.args = {
-  variant: 'Navigation',
+export const Navigation: StoryObj<typeof Button> = {
+  args: {
+    variant: 'Navigation',
+  },
 }
 
-export const Overlay: Story<ButtonProps> = DefaultStory.bind({})
-Overlay.args = {
-  variant: 'Overlay',
+export const Overlay: StoryObj<typeof Button> = {
+  args: {
+    variant: 'Overlay',
+  },
 }
 
-export const Danger: Story<ButtonProps> = DefaultStory.bind({})
-Danger.args = {
-  variant: 'Danger',
+export const Danger: StoryObj<typeof Button> = {
+  args: {
+    variant: 'Danger',
+  },
 }
 
-export const Small: Story<ButtonProps> = DefaultStory.bind({})
-Small.args = {
-  size: 'S',
+export const Small: StoryObj<typeof Button> = {
+  args: {
+    size: 'S',
+  },
 }
 
-export const Fixed: Story<ButtonProps> = DefaultStory.bind({})
-Fixed.args = {
-  fullWidth: true,
+export const Fixed: StoryObj<typeof Button> = {
+  args: {
+    fullWidth: true,
+  },
 }
 
-export const Disabled: Story<ButtonProps> = DefaultStory.bind({})
-Disabled.args = {
-  disabled: true,
+export const Disabled: StoryObj<typeof Button> = {
+  args: {
+    disabled: true,
+  },
 }
 
-export const Link: Story<ButtonProps> = DefaultStory.bind({})
-Link.args = {
-  to: '#',
+export const Link: StoryObj<typeof Button> = {
+  args: {
+    to: '#',
+  },
 }
 
-const NihongoStory = (args: ButtonProps) => (
-  <Button {...args} onClick={action('click')}>
-    日本語だよ
-  </Button>
-)
-
-export const Nihongo: Story<ButtonProps> = NihongoStory.bind({})
-
-const FocusStory = (args: ButtonProps) => <FocusStoryInternal {...args} />
-const FocusStoryInternal = (args: ButtonProps) => {
-  const ref = useRef<ClickableElement>(null)
-  const focus = () => ref.current?.focus()
-  return (
-    <Button {...args} onMouseOver={focus} ref={ref}>
-      Mouse over to focus
+export const Nihongo: StoryObj<typeof Button> = {
+  render: (args) => (
+    <Button {...args} onClick={action('click')}>
```

Comment:
Since we're using @storybook/addon-essentials, I would suggest that onClick={action('click')} may be unnecessary.